fpo-api/api/migrations/0004_auto_20200908_1557.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 2.1.15 on 2020-09-08 22:57
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0003_surveyresult'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Application',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('app_type', models.CharField(blank=True, default='', max_length=100)),
+                ('last_updated', models.DateTimeField(blank=True, null=True)),
+                ('current_step', models.CharField(blank=True, default='', max_length=100)),
+                ('current_page', models.CharField(blank=True, default='', max_length=100)),
```

Comment:
the "current_page" field is part of the step table. the application table requires the fields listed in the ApplicationState interface in "fpo-web/src/models/storeState.ts".